native/templates/windows/main.cpp

```diff
@@ -26,8 +26,17 @@
 #include <iostream>
 
 #include "platform/BasePlatform.h"
+
+#if defined(CC_SERVER_MODE)
+#include <windows.h>
+int WINAPI
+WinMain(HINSTANCE hInst, HINSTANCE hPrev, LPSTR szCmdLine, int sw) {
+    START_PLATFORM(0, nullptr);
+}
+#else
```

Comment:
Please don't modify templates. It will break compatibility.

Reply:
because this use sdl main， server mode no link sdl libs

Reply:
Ok, as it doesn't affect normal projects, so not problem.